homeassistant/components/sensor/zha.py

```diff
@@ -111,3 +131,155 @@ def state(self):
             return 'unknown'
 
         return round(float(self._state) / 100, 1)
+
+
+class GenericBatterySensor(Sensor):
+    """ZHA generic battery sensor."""
+
+    value_attribute = 32
+    battery_sizes = {
```

Comment:
This should go into zigpy as an enum, as it is part of the standard.

Reply:
I'll submit a PR to zigpy for this